blog/2022-06-30-paralus-quickstart-aks/index.md

```diff
@@ -0,0 +1,200 @@
+- [Using the Azure Portal](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-powershell)
+- [Using the ARM template](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-rm-template)
+
+You are free to choose any option that you want, we used the Azure Portal to create one.
+
+> Note: Choose a virtual machine with atleast 4 vCPUs and 16 GB of RAM. Refer to the list of [Azure VMs](https://azure.microsoft.com/en-in/pricing/details/virtual-machines/linux/).
+
+After the cluster is created, start the cluster and connect to it. If you've created a cluster using the Azure portal, you can use the Azure Cloud Shell to connect to the cluster.
+
+## Installing Paralus
+
+Clone the [paralus helm repository](https://github.com/paralus/helm-charts)
+
+1. Add helm repo
+
+   `helm repo add paralus https://paralus.github.io/helm-charts`
+
+2. Create `values.aks.yaml`
+
+   ```yaml
+   ingress:
+     enabled: false
+
+   deploy:
+     elasticsearch:
+       enable: true
+     postgresql:
+       enable: true
+     contour:
+       enable: true
+
+   kratos:
+     kratos:
+       development: true
+   ```
+
+   > Note: When deploying this in production, set `kratos.development: false`
+
+3. Create `values.domain.yaml`
+
+   ```yaml
+   fqdn:
+     domain: "chartexample.com"
+     hostname: "console-aks-oss"
+     coreConnectorSubdomain: "*.core-connector.aks-oss"
+     userSubdomain: "*.user.aks-oss"
+   ```
+
+4. Install Paralus
+
+   ```bash
+   helm install myrelease paralus/ztka --devel -f values.domain.yaml -f values.aks.yaml -n paralus --create-namespace
+
+   NAME: myrelease
+   LAST DEPLOYED: Wed Jun 29 10:13:48 2022
+   NAMESPACE: paralus
+   STATUS: deployed
+   REVISION: 1
+   NOTES:
+   1. Access the application URL by running these commands:
+     Get load balancer address via:
+     kubectl get service envoy --namespace paralus -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'
+
+     Add DNS records of following domains such that it resolves to above address:
+     - console-aks-oss.chartexample.com
+     - *.core-connector.aks-oss.chartexample.com
+     - *.user.aks-oss.chartexample.com
+     Open http://console-aks-oss.chartexample.com in browser.
+
+   You can view the recovery link for admin user by running the following command once all the pods are running:
+
+   kubectl logs -f --namespace paralus $(kubectl get pods --namespace paralus -l app.kubernetes.io/name='paralus' -o jsonpath='{ .items[0].metadata.name }') initialize-paralus | grep 'Org Admin signup URL:'
+   ```
+
+   > Note: It can take upto a few minutes before all the pods are running and you can access the dashboard. You can check the status using `watch kubectl get pods`
+
+## Configuring DNS Settings
+
+Once the installation is complete, you need to first get the external IP address provided by Azure loadbalancer. You can do so by executing the following command:
+
+```bash
+kubectl get svc -n paralus
```

Comment:
`kubectl get svc myrelease-contour-envoy -n paralus` is more specific command. what do you think?

Reply:
Done